clio-server/src/main/scala/org/broadinstitute/clio/server/service/ServerService.scala

```diff
@@ -118,6 +118,7 @@ class ServerService private[server] (
     val indexes = immutable.Seq(
       ElasticsearchIndex.Ubam,
       ElasticsearchIndex.Gvcf,
+      ElasticsearchIndex.Bam,
```

Comment:
Not that it matters much, but is there a reason for this ordering?

Reply:
It was just for my own use always having the almost identical Cram stuff close by when implementing.